src/init.c

```diff
@@ -60,7 +60,9 @@ extern void _conn_mock_fini();
 
 extern void _chunk_append_init();
 
+#if PG16_LT
 extern void TSDLLEXPORT _PG_init(void);
+#endif
```

Comment:
I wasn't aware that this was removed in PG16. What is the method to run initialization in PG16 and later?